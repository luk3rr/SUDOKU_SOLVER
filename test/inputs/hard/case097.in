300001052 005300000 009060830 057040000 800000095 000200400 010809000 700050060 030010000 
