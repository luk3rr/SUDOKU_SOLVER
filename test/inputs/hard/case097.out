Solving the following grid:
3 0 0 | 0 0 1 | 0 5 2 
0 0 5 | 3 0 0 | 0 0 0 
0 0 9 | 0 6 0 | 8 3 0 
------+-------+------
0 5 7 | 0 4 0 | 0 0 0 
8 0 0 | 0 0 0 | 0 9 5 
0 0 0 | 2 0 0 | 4 0 0 
------+-------+------
0 1 0 | 8 0 9 | 0 0 0 
7 0 0 | 0 5 0 | 0 6 0 
0 3 0 | 0 1 0 | 0 0 0 


Solution found :')

3 8 4 | 7 9 1 | 6 5 2 
6 2 5 | 3 8 4 | 9 7 1 
1 7 9 | 5 6 2 | 8 3 4 
------+-------+------
2 5 7 | 9 4 8 | 3 1 6 
8 4 3 | 1 7 6 | 2 9 5 
9 6 1 | 2 3 5 | 4 8 7 
------+-------+------
5 1 6 | 8 2 9 | 7 4 3 
7 9 2 | 4 5 3 | 1 6 8 
4 3 8 | 6 1 7 | 5 2 9 

Algorithm: BFS
Total time: 777 ms
Total expanded states: 100928
